src/tequilapi-client.ts

```diff
@@ -539,4 +545,9 @@ export class HttpTequilapiClient implements TequilapiClient {
   public async getReferralToken(identity: string): Promise<ReferralTokenResponse> {
     return this.http.get(`/identities/${identity}/referral`)
   }
+
+  public async exchangeRate(quoteCurrency = 'usd'): Promise<Money> {
+    const baseCurrency = 'myst'
+    return this.http.get(`/exchange/${baseCurrency}/${quoteCurrency}`)
```

Comment:
The endpoint only allows `myst` as baseCurrency so I guess you could just add it as part of the string 🤷🏻

Reply:
It does not accept the base currency as an argument, so the variable is just for documentation purposes.